src/OrchardCore.Modules/OrchardCore.ContentPreview/ResourceManifest.cs

```diff
@@ -0,0 +1,18 @@
+using OrchardCore.ResourceManagement;
+
+namespace OrchardCore.ContentPreview
+{
+    public class ResourceManifest : IResourceManifestProvider
+    {
+        public void BuildManifests(IResourceManifestBuilder builder)
+        {
+            var manifest = builder.Add();
+
+            manifest
+                .DefineScript("contentpreview-edit")
+                .SetDependencies("admin")
```

Comment:
I see now @ns8482e when I look closer.

So for front end preview the dependency is jQuery (because we wouldn't want to include the admin js on the front end), and for the template module it is admin.

I think it could just be jquery for both, but you would need to test it (or I maybe wrong) - and check the loading order, to make sure.

Or we just go back to how you had it `<script asp-name='prev' depends-on="admin/jquery">

I'm ok either way

Reply:
Either way is fine for me, as it's replacing following line - only applicable in `/admin`

`<script asp-src="~/OrchardCore.ContentPreview/Scripts/contentpreview.edit.js" depends-on="admin" at="Foot"></script>`

to named script as below

`<script asp-name="contentpreview-edit" at="Foot"></script>`

Originally i kept `depends-on` in `cshtml` thinking the possibility that admin theme can be changed, and/or when template is overridden.

Reply:
Cool, let me check again tomorrow. I think my comment was wrong.

For infos. 
We always use `admin` as the script and style name (because we do everywhere), and a custom admin theme should still register its own script and style named `admin`. Then there's no problem, or conflict, and things just keep working.